Cogs.Publishers/JsonSimpleConverter.cs

```diff
@@ -36,7 +36,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
                 obj.Add(new JProperty("time", 
                     new JObject(
                         new JProperty("type", "string"), 
-                    new JProperty("format", "time"))));
+                    new JProperty("pattern", "(00|[0-9]|1[0-9]|2[0-3]):([0-9]|[0-5][0-9]):([0-9]|[0-5][0-9])+Z"))));
```

Comment:
the time zone is defined as Z or a time-numoffset which looks like +06:00

```
time-offset     = "Z" / time-numoffset
time-numoffset  = ("+" / "-") time-hour ":" time-minute
```

https://tools.ietf.org/html/rfc3339#section-5.6